lib/ReactCrop.js

```diff
@@ -713,7 +713,7 @@ class ReactCrop extends PureComponent {
             <div className="ReactCrop__drag-handle ord-w" data-ord="w" />
           </div>
         )}
-        {renderSelectionAddon && (
+        {renderSelectionAddon && !!crop.width && (
```

Comment:
Thanks for this PR, just to be a bit more semantic can you please change this to `&& this.isCropValid(crop)`

Reply:
Updated already